sig/lrama/grammar/code/rule_action.rbs

```diff
@@ -0,0 +1,19 @@
+module Lrama
+  class Grammar
+    class Code
+      class RuleAction < Code
+        @rule: Rule
+
+        def initialize: (type: ::Symbol, token_code: Grammar::Code, rule: Rule) -> void
+
+        private
+
+        def reference_to_c: (Reference ref) -> String
+        def position_in_rhs: () -> Integer
+        def rhs: () -> Array[Grammar::Symbol]
+        def lhs: () -> Symbol
```

Comment:
Is there any intention to use "Symbol" however `rhs` is `Grammar::Symbol`.

Reply:
Sorry, seems like a simple mistake. I fixed.